File: Go_api/Models/wishlistitem.go
package models

import (
	"gorm.io/gorm"
)

type WishlistItem struct {
	gorm.Model
	WishlistID uint
	Wishlist   Wishlist `gorm:"foreignKey:WishlistID"`
	ProductID  uint
	Product    Product `gorm:"foreignKey:ProductID"`
}

File: Django_api/backend/models/wishlist_item.py
from django.db import models
from django.conf import settings
from .wishlist import Wishlist
from .product import Product

class WishlistItem(models.Model):
    wishlist = models.ForeignKey(Wishlist, on_delete=models.CASCADE)
    product = models.ForeignKey(Product, on_delete=models.CASCADE)
